.github/ISSUE_TEMPLATE/chore.yml

```diff
@@ -33,7 +33,7 @@ body:
         - unit tests added/updated;
         - integration/compatibility tests added/updated;
         - spot refactorings done;
-        - user documentation updated;
+        - user documentation updated or issue to update documentation created;
```

Comment:
The tricky part is that currently, we have this definition of done list for `chore` tasks only, but quite often such issues about user documentation need to be created when we implement new features. So, let me know if you see what we can do about it.

Reply:
I think we briefly talked that we can create a chore task, then change it to other task type.